doc/qbk/1.0.overview.qbk

```diff
@@ -14,24 +15,35 @@
 
 Boost.URL is a portable C++ library which provides containers and algorithms
 which model a "URL", more formally described using the
-[@https://https://datatracker.ietf.org/doc/html/rfc3986 Uniform Resource Identifier (URI)]
+[@https://datatracker.ietf.org/doc/html/rfc3986 Uniform Resource Identifier (URI)]
 specification (henceforth referred to as __rfc3986__). A URL is a compact sequence
 of characters that identifies an abstract or physical resource. For example,
 this is a valid URL which satisfies the ['absolute-URI] grammar:
+
+[teletype]
```

Comment:
teletype means what's coming is plain text and it should try to highlight it as C++